dist/debian/control.template

```diff
@@ -17,7 +17,7 @@ Conflicts: %{product}-server (<< 1.1)
 
 Package: %{product}-server
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, %{product}-conf (= ${binary:Version}), %{product}-python3 (= ${binary:Version})
+Depends: ${shlibs:Depends}, ${misc:Depends}, %{product}-conf (= ${binary:Version}), %{product}-python3 (= ${binary:Version}), %{product}-node-exporter (= ${binary:Version})
```

Comment:
Isn't it optional? Better to make just scylla depend on it, not scylla-server, so people can do minimal installs.

Reply:
Okay, then I'll change scylla_setup code to prevent traceback when scylla-node-exporter is not installed(#8276), instead of changing dependency.

Reply:
Pushed a fix for scylla_setup.